packages/cli/src/commands/start/startFrontend.ts

```diff
@@ -32,6 +32,30 @@ interface StartAppOptions {
   configPaths: string[];
 }
 
+function checkReactVersion() {
+  try {
+    // Make sure we're looking at the root of the target repo
+    const reactPkgPath = require.resolve('react/package.json', {
+      paths: [paths.targetRoot],
+    });
+    const reactPkg = require(reactPkgPath);
+    if (reactPkg.version.startsWith('16.')) {
+      console.log(
+        chalk.yellow(
+          `
+⚠️                                                                           ⚠️
+⚠️ You are using React version 16, which is deprecated for use in Backstage. ⚠️
+⚠️ Please upgrade to at React 17 by updating your packages/app dependencies. ⚠️
```

Comment:
```suggestion
⚠️ Please upgrade to at least React 17 by updating your packages/app dependencies. ⚠️
```

Reply:
well at least (see what i did there?) remove the word "at" then.

Reply:
Derp!